### PR TITLE
Edit button

### PR DESCRIPTION
**Pull request by Changron**
1. add edit button in contest.html

### CODE REVIEW

File: contest/templates/contest/contest.html [hunk 1]
```diff
@@ -14,7 +14,16 @@
   <div class="container">
     <!--Title-->
     <div class="well">
-      <h2>{{ contest.cname }}</h2>
+      <h2>{{ contest.cname }}
+      {% if user|can_edit_contest:contest %}
+       <a href="{% url 'contest:edit' contest.id %}">
+        <button type="button" class="btn btn-warning btn-sm" data-toggle="tooltip"
+              data-placement="top" title="Edit Contest">
```
Comment (nickelcheng) [inline contest/templates/contest/contest.html]: I prefer ```data-placement``` align with ```type```

**Comment by nickelcheng** [comment]
+1 besides the comment.

**Comment by Changron** [comment]
fixed

**Comment by bruce3557** [comment]
+1, going to merge

**Pull request merged**